client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -301,6 +323,24 @@ def _create_artifact_msg(
         )
         return artifact_msg
 
+    @classmethod
+    def _pretty_artifact_str(cls, artifact_msg):
```

Comment:
Since `cls` goes unused, I'd recommend going with a `staticmethod` in the interesting of being persimmonious.

```suggestion
    @staticmethod
    def _pretty_artifact_str(artifact_msg):
```